docs/tutorials/agent_tutorial.rst

```diff
@@ -32,6 +32,7 @@ First, we define the constructor:
             self._q_values = np.zeros(obs_dim, act_dim)
             self._gamma = gamma
             self._alpha = alpha
+            self._act_dim = act_dim
             self._epsilon_schedule = LinearSchedule(1.0, final_epsilon, explore_steps)
 
```

Comment:
This tutorial needs to be updated based on the new api. We no longer pass act_dim and obs_dim, we instead pass spaces. Can you please update the tutorial?